packages/core/content-type-builder/admin/src/components/DraftAndPublishToggle/index.js

```diff
@@ -66,65 +56,28 @@ const DraftAndPublishToggle = ({
 
   return (
     <>
-      <ToggleInput
-        checked={value || false}
-        disabled={disabled}
-        hint={hint}
-        label={label}
-        name={name}
-        offLabel={formatMessage({
-          id: 'app.components.ToggleCheckbox.off-label',
-          defaultMessage: 'Off',
-        })}
-        onLabel={formatMessage({
-          id: 'app.components.ToggleCheckbox.on-label',
-          defaultMessage: 'On',
-        })}
-        onChange={handleChange}
+      <Checkbox checked={value} disabled={disabled} hint={hint} name={name} onChange={handleChange}>
+        {label}
+      </Checkbox>
+
+      <ConfirmDialog
+        isOpen={showWarning}
+        onToggleDialog={handleToggle}
+        onConfirm={handleConfirm}
+        bodyText={{
+          id: getTrad('popUpWarning.draft-publish.message'),
+          defaultMessage:
+            'If you disable the draft & publish, your drafts will be deleted. Are you sure you want to disable it?',
+        }}
```

Comment:
Modified to match the translation string and made a tiny wording change. This makes more sense to me.
```suggestion
        bodyText={{
          id: getTrad('popUpWarning.draft-publish.message'),
          defaultMessage: 'If you disable draft & publish, your drafts will be deleted.',
        }}
```

Reply:
Done in 35bbe83248ddbef66864e04f6ec12ef498ffb022